frontend/public/locales/en/translation.json

```diff
@@ -2747,6 +2749,8 @@
   "status.unknown": "Unknown",
   "status.unknown.alert.title": "The cluster is not reachable",
   "status.unknown.message": "The cluster is not reachable. The management control plane has not heard from the cluster in the specified <bold>leaseDurationSeconds</bold> interval configured in the <bold>ManagedCluster</bold> resource.",
+  "status.unreachable": "Unreachable",
+  "status.unreachable.message": "The cluster is not reachable. The management control plane has not heard from the cluster in the specified <bold>leaseDurationSeconds</bold> interval configured in the <bold>ManagedCluster</bold> resource.",
```

Comment:
This description is not quite accurate. The "Unreachable" status is not caused by the control plane not receiving information from the managed cluster, but rather because the control plane has been unable to access the managed cluster's API server for an extended period of time.